conan/cli/cli.py

```diff
@@ -32,20 +32,26 @@ def __init__(self, conan_api):
         self._conan_api = conan_api
         self._groups = defaultdict(list)
         self._commands = {}
+
+    def _add_commands(self):
         conan_commands_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "commands")
         for module in pkgutil.iter_modules([conan_commands_path]):
             module_name = module[1]
             self._add_command("conan.cli.commands.{}".format(module_name), module_name)
 
-        custom_commands_path = ClientCache(conan_api.cache_folder).custom_commands_path
+        custom_commands_path = ClientCache(self._conan_api.cache_folder).custom_commands_path
         if not os.path.isdir(custom_commands_path):
             return
 
         sys.path.append(custom_commands_path)
         for module in pkgutil.iter_modules([custom_commands_path]):
             module_name = module[1]
             if module_name.startswith("cmd_"):
-                self._add_command(module_name, module_name.replace("cmd_", ""))
+                try:
+                    self._add_command(module_name, module_name.replace("cmd_", ""))
+                except Exception as e:
+                    raise ConanException("Error loading custom command "
```

Comment:
I think it would be better to write an error, but continue. The problem with this is that a defective command won't allow you to get a fix for it with ``conan config install`` for example. If the skipped command is used later, then it will fail, no prob.